editor/editor_run_native.cpp

```diff
@@ -135,8 +135,8 @@ Error EditorRunNative::start_run_native(int p_id) {
 		bool is_arch_enabled = preset->get(preset_arch);
 
 		if (!is_arch_enabled) {
-			String warning_message = vformat(TTR("Warning: The CPU architecture \"%s\" is not active in your export preset.\n\n"), Variant(architecture));
-			warning_message += TTR("Run \"Remote Deploy\" anyway?");
+			String warning_message = vformat(TTR("Warning: The CPU architecture \"%s\" is not active in your export preset."), Variant(architecture));
+			warning_message += "\n\n" + TTR("Run \"Remote Deploy\" anyway?");
 
 			run_native_confirm->set_text(warning_message);
```

Comment:
```suggestion
			run_native_confirm->set_text(vformat(TTR("Warning: The CPU architecture \"%s\" is not active in your export preset.\n\nRun \"Remote Deploy\" anyway?"), architecture));
```

Separating into two strings is usually done for the purpose of reusing translations. However, both strings here are unique so they can be combined into one.

This gives more context to the translators and also makes it consistent with other similar dialogs.

Reply:
There may actually be a few more cases like this. The 4.4 project warning to generate UID files comes to mind.